examples/basic/index.html

```diff
@@ -75,7 +78,10 @@
       },
       'vive-controls': {
         gripdown: 'changeTask',
-        trackpaddown: 'logdefault'
+        trackpaddown: 'logdefault',
+        'trackpad.doubletouch': 'doubletouch',
```

Comment:
Can we remove the dot for consistency with other event names? `trackpaddoubletouch`, `'trackpaddoublepress', ` 'trackpadlongpress'